client/app/scripts/liveblog-edit/module.js

```diff
@@ -135,6 +137,11 @@ define([
                 $scope.panelState = panel;
                 // update url for deeplinking
                 $route.updateParams({panel: $scope.panelState});
+                //clear the new contribution notification
+                if (panel === 'contributions') {
+                    postsService.resetUnreadContributtions(panel);
+                }
+                postsService.setPanelState(panel);
```

Comment:
This instruction is redundant with `$scope.panelState = panel;` L137,
and more generally, the posts service shouldn't be aware of the panels.
Panels are graphical elements, that only controllers should be aware of.